src/parse/rules/proto_rules.py

```diff
@@ -20,6 +20,10 @@
 }
 
 
+DEFAULT_PROTO_LABELS = ['protobuf']
```

Comment:
We already have this, but it's called "proto" - try e.g. `plz query alltargets --include proto --hidden`. Adding grpc separately is probably useful though.